backend/python/app/services/implementations/story_service.py

```diff
@@ -135,20 +142,35 @@ def get_story_translation(self, id):
                     Story.description.label("description"),
                     Story.youtube_link.label("youtube_link"),
                     Story.level.label("level"),
+                    StoryTranslation.id.label("id"),
+                    StoryTranslation.language.label("language"),
+                    StoryTranslation.stage.label("stage"),
+                    StoryTranslation.translator_id.label("translator_id"),
+                    StoryTranslation.reviewer_id.label("reviewer_id"),
+                    StoryTranslationContent.id.label("content_id"),
+                    StoryTranslationContent.line_index.label("line_index"),
+                    StoryTranslationContent.translation_content.label("translation_content"),
                 )
                 .join(StoryTranslation, Story.id == StoryTranslation.story_id)
+                .join(StoryTranslationContent, StoryTranslationContent.story_translation_id == StoryTranslation.id)
                 .filter(StoryTranslation.id == id)
-                .one()
-                ._asdict()
+                .all()
             )
 
-            story_details["num_translated_lines"] = self._get_num_translated_lines(
-                translation.translation_contents
+            response = {**story_details[0]._asdict()}
+            response.pop("content_id")
```

Comment:
Rather than pop-ing these attributes, what about removing lines 150-152?

Reply:
Lines 150-152 are required as each result from `StoryTranslationContent` in the `translationContent` part of the gql query needs those attributes. Popping them is just to create the initial dict that needs to be returned, with the for loop creating the list of dicts for the `translationContent` field

Reply:
Interesting, to clarify, what is at `response["content_id"]` and `response["line_index"]` and `response["translation_content"]` at this point of the code? Is it just empty? looking at this function again, I'm surprised these fields exist on the response

Reply:
It would just be the values for the first result of the sql query (ie. the details for the first translated line of specified story_translation) since the returned response needs all the other key-value pairs and then needs lines 150-152 as a list of dicts for the `translationContent` field